### PR TITLE
Add ST7920 doc

### PR DESCRIPTION
**Pull request by marsjan155**
## Description:

Docs for ST7920 LCD Display

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#1440

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by frenck — APPROVED** [review]
@marsjan155, A wild merge conflict appeared, out of nowhere! 🙈 

Could you please rebase this PR onto the latest `next` branch and resolve the merge conflict?

Thanks already! 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]
Now with the correct review status :)

**Comment by marsjan155** [comment]
@frenck
rebased and reverted one unnecessary change, should be fine, thanks

File: components/display/st7920.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+displays with ESPHome. Note that this component is for displays that are connected via the 3-Wire :ref:`SPI bus <spi>`.
+It's a monochrome LCD graphic display.
+
+.. figure:: images/st7920-full.jpg
+    :align: center
+    :width: 75.0%
+
+    ST7920 LCD Graphic Display
+
+Connect D5 to the E pin you chose for the :ref:`SPI bus <spi>`, connect D7 to the R/W pin and ``RS`` to some GPIO pins one the ESP.
```
Comment (frenck) [inline components/display/st7920.rst]:
```suggestion
Connect D5 to the E pin you chose for the :ref:`SPI bus <spi>`, connect D7 to the R/W pin and ``RS`` to some GPIO pins on the ESP.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**